test/cpp/api/modules.cpp

```diff
@@ -1451,7 +1453,7 @@ TEST_F(ModulesTest, FunctionalWithTorchFunction) {
 
 TEST_F(ModulesTest, FunctionalArgumentBinding) {
   auto functional =
-      Functional(torch::elu, /*alpha=*/1, /*scale=*/0, /*input_scale=*/1);
+      Functional(static_cast<Tensor(*)(const Tensor&, Scalar, Scalar, Scalar)>(torch::elu), /*alpha=*/1, /*scale=*/0, /*input_scale=*/1);
```

Comment:
Per the PR description, this potentially kind of a rough change if people are using this idiom much. Did we consider using a different suffix for faithful-out? It's ugly but I'm not sure that matters.